src/index.js

```diff
@@ -17,7 +17,7 @@ async function translate(text, options) {
 
     // Check if a lanugage is in supported; if not, throw an error object.
     let error;
-    [ options.from, options.to ].forEach((lang) => {
+    [options.from, options.to].forEach((lang) => {
```

Comment:
probably you should revert this whitespace changes. in this PR we expect to see changes related to subject of this PR only.

any whitespace/formatting issues may be addressed in separate PR